### PR TITLE
add star class

### PR DESCRIPTION
**Pull request by rachel-ellis**

### CODE REVIEW

File: app/src/main/java/com/example/labparticipation/Star.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package com.example.labparticipation;public class Star {
```
Comment (nic3-14159) [inline app/src/main/java/com/example/labparticipation/Star.java]: Add newline before `public class`

**Review by nic3-14159 — CHANGES_REQUESTED** [review]

**Review by ekhumbata — APPROVED** [review]
NO YOU'RE A STAR!!!

File: app/src/main/java/com/example/labparticipation/Star.java (hunk 1 above)
Comment (rachel-ellis) [reply app/src/main/java/com/example/labparticipation/Star.java]: oh weird, it looks fine in Android Studio 
Screen Shot 2023-01-31 at 5 54 34 PM

Comment (rachel-ellis) [reply app/src/main/java/com/example/labparticipation/Star.java]: nvm should be good for a re-review now lol

**Review by nic3-14159 — APPROVED** [review]
lgtm

**Pull request merged**